contrib/win32/openssh/build.json

```diff
@@ -11,7 +11,7 @@
       "DockerFile": ".\\contrib\\win32\\openssh\\DockerFile",
       "AdditionalContextFiles": [ ".\\contrib\\win32\\openssh\\OpenSSH-build.ps1" ],
       "DockerImageName": "di-openssh-x64",
-      "BinaryBucket": "OpenSSH-Win64",
+      "BinaryBucket": "OpenSSH-Original",
       "ArtifactsExpected": 1,
```

Comment:
Set version to 7.7.1.1.. where are we hosting these changes?

Reply:
This is the VSTS artifacts folder name.. If we have the version number then we have to change it everytime when we release a new binary,

Reply:
I understand :). I just chose it as a placeholder for version change comment as that file was not included in prior PR.